### PR TITLE
LRQA-68075 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-68075

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -1812,7 +1812,7 @@ definition {
 	}
 
 	@priority = "4"
-	test ExportAllEntries {
+	test ExportMultipleEntries {
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:1815]: Name it in SACRED model.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @melodywu,
Not sure if I missing any further communication based on the issue, correct me if I did, but based on the description in the related issue, 

> Rename GDPR#ExportAllEntries to GDPR#CanExportMultipleEntries
> Rename existing tests

We should pay attention to following places,
1. Name the test in SACRED model. ExportAllEntries to **Can**ExportMultipleEntries
2. Rename the all entries to multiple entries, ExportAllEntries to **Can**Export**Multiple**Entries
3. Rename other existing test based on above principles, update the test name not only for those contains all entries but also for those name not in SACRED model

**Comment by melodywu** [comment]
@Sharryxue  I list the names. Is there anyone I need to update?
CanAnonymizeAllEntriesFromAnApplication
CanAnonymizeARelatedAsset
CanAnonymizeASingleStagedAndLiveWebContentEntry
CanAnonymizeEntriesFromAnApplicationWithHierarchy
CanAnonymizeEntryFromApplication
CanAnonymizeMultipleEntries
CanAnonymizeMultipleEntriesFromAnApplication
CanAnonymizeMultipleStagedDataFromDM
CanAnonymizeViaInfoPanel
CanAppsWithNoEntriesAreDisabledInNewDataExport

CanDeleteAndAnonymizeMultipleEntries
CanDeleteAnExportProcess
CanDeleteARelatedAsset
CanDeleteASingleStagedAndLiveBlogsEntry
CanDeleteEntriesFromAnApplicationWithHierarchy
CanDeleteEntryFromApplication
CanDeleteMultipleEntriesFromAnApplication
CanDeleteMultipleEntriesFromInstanceScope
CanDeleteMultipleEntriesFromPersonalSiteScope
CanDeleteMultipleEntriesFromRegularSitesScope
CanDeleteMultipleStagedDataFromAnApplication
CanDisplayEntryDetailsInInfoPanel

CanEditEntryFromApplication
CanExportMultipleEntries
CanFilterAndOrderExportProcesses 
CanFilterAndViewData
CanOrderDataInViewData
CanPublishToLiveADeletedLiveEntry
EntriesAreStillAnonymousAfterActivatingUser
PaginationWorksInExportPersonalData
PaginationWorksInNewDataExport
RemainingItemsCountIsAccurate

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Thanks, @melodywu! For next time, I think it would be nice to separate these into different commits to make it easy to review. Please see https://github.com/patriciajeanperez/liferay-portal/pull/737.

**Comment by melodywu** [comment]
> Thanks, @melodywu! For next time, I think it would be nice to separate these into different commits to make it easy to review. Please see #737.

Ok， thank you Patty！

**Pull request closed without merging**